llama_hub/llama_packs/panel_chatbot/app.py

```diff
@@ -0,0 +1,371 @@
+"""Provides a ChatBot UI for a Github Repository. Powered by Llama Index and Panel"""
```

Comment:
I also contribute to https://holoviz-topics.github.io/panel-chat-examples/. And due to limits to how much time I can spend :-), I have to make a choice whether LLamaIndex examples should live here or at panel-chat-examples.

I would prefer to contribute them here and over time turn the `PanelChatPack` into a multi page app with a file for each page. And maybe even a `model.py` or `component.py` file containing reusable components for the most used LLamaIndex classes.

That would bring them close to the users. The I would just link from Panel-Chat-Examples to this location.

What do you think about that idea?

Reply:
@MarcSkovMadsen that sounds great. we can promote it from our page and hopefully give you eyeballs/downloads from llamahub. and of course would appreciate a backlink from your docs page!

Reply:
Thx.